pkg/kubectl/cmd/certificates/certificates.go

```diff
@@ -47,13 +48,13 @@ func NewCmdCertificate(f cmdutil.Factory, ioStreams genericclioptions.IOStreams)
 		},
 	}
 
-	cmd.AddCommand(NewCmdCertificateApprove(f, ioStreams))
-	cmd.AddCommand(NewCmdCertificateDeny(f, ioStreams))
+	cmd.AddCommand(newCmdCertificateApprove(f, ioStreams))
+	cmd.AddCommand(newCmdCertificateDeny(f, ioStreams))
 
 	return cmd
 }
 
-type CertificateOptions struct {
```

Comment:
doc rather than unexport

Reply:
hi @liggitt 
Thank you help to review my code

This structure is only referenced in this package, and changing the initial letter to lowercase is also a solution.

Best Wishes

Reply:
The intent behind all of these commands is for them to be reusable. Please do not unexport the options or constructors.

Reply:
OK
Thanks @liggitt

I will revise it according to your request for adding godoc for these。

Best Wishes